src/modules/SdnDiag.Utilities/private/Trace-Exception.ps1

```diff
@@ -22,5 +22,5 @@ function Trace-Exception {
         $Exception
     )
 
-    Trace-Output -Level:Error -Message $Exception.Exception.Message -Exception $Exception
+    Trace-Output -Exception $Exception -FunctionName (Get-PSCallStack)[1].Command -Level 'Exception'
```

Comment:
-FunctionName should have been default to `(Get-PSCallStack)[1].Command` is this duplicate?

Reply:
By the time it enters into `Trace-Output`, it becomes `Get-PSCallStack)[2].Command` which is why I am manually specifying it within the `Trace-Exception` command. Basically it's the caller of function that [1] would indicate.